### ISSUE TITLE
missing support for MFA

### ISSUE THREAD

**Issue by vjirovsky** [lkavale/LuKaSo.Zonky#1]
hi, there is missing implementation for MFA - so effectively from 15-Sep-2019 will SDK probably stop to work...

reference for MFA on Zonky API docs:
https://zonky.docs.apiary.io/#reference/oauth/token/get-access-token

I will try to find some time to send PR, if you are not developing it already

**Comment by lkavale** [comment]
Hi, yes you are right, when Zonky starts require 2FA, this library will fail. I am happy to see that they update their API spec on apiary.com, it is very unusual... Usually they not publishing their changes in API. So I planed some updates after 15th when will be possible see the changes on web. But now, when you sent me the spec I am possible to implements changes earlier. This requires updates of log-in methods and implementing the presistent authorization of the client.

**Comment by vjirovsky** [comment]
There is temporary phase - user needs to log in on web (via required 2FA) and until end of September is not 2FA required for API

**Comment by lkavale** [comment]
Yes, old version of library is still working. I made some changes, but I was unable to test it against zonky API. The API never requests me to put sms code. This week I will finish some changes in persistent authorization mechanism, this version should be ready for 2FA.

**Comment by lkavale** [comment]
2FA is required only for exporting account transactions (GetWalletTransactionsAsync, if you want ) and for transferring money to bank account (as always). Money transfers are not supported in this library, so only exports of account transactions needs the MFA. Ineed some additional testing, this week will be released new version with MFA support.
